CONTRIBUTORS.md

```diff
@@ -1,16 +1,16 @@
+| [<img src="https://avatars0.githubusercontent.com/u/2049543?v=4" width="100px;"/><br /><sub><b>Willian Taiguara Baliza Moura</b></sub>](https://github.com/willianwt)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=willianwt "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=willianwt "Code") | [<img src="https://avatars3.githubusercontent.com/u/33730790?v=4" width="100px;"/><br /><sub><b>Rajashree Parhi</b></sub>](https://github.com/rajashree23)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=rajashree23 "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=rajashree23 "Code") | [<img src="https://avatars1.githubusercontent.com/u/40717067?v=4" width="100px;"/><br /><sub><b>Akshit Aggarwal</b></sub>](https://github.com/AkshitAggarwal)<br />[📖](https://github.com/CodeToExpress/dailycodebase/commits?author=AkshitAggarwal "Documentation") [💻](https://github.com/CodeToExpress/dailycodebase/commits?author=AkshitAggarwal "Code") | [<img src="https://avatars1.githubusercontent.com/u/44455626?v=4" width="100px;"/><br /><sub><b>Abhishek</b></sub>](https://github.com/dasabhishek0790)<br />[💻](https://github.com/CodeToExpress/dailycodebase/commits?author=dasabhishek0790 "Code") |
```

Comment:
The details of @sagunjaiswal  have been removed from the Markdown file. Can you add them back to the file?

Reply:
@aaditkamat  : Added, not sure how the contributors file got changed. Please review & merge.